src/frontend/components/InstructorDashboardVOD/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import React from 'react';
+import styled from 'styled-components';
+
+import { DashboardVideoLiveWidgetsContainer } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetsContainer';
+import { TeacherLiveInfoBar } from 'components/TeacherLiveInfoBar';
+import VideoPlayer from 'components/VideoPlayer';
+import { useCurrentVideo } from 'data/stores/useCurrentRessource/useCurrentVideo';
+import { InfoWidgetModalProvider } from 'data/stores/useInfoWidgetModal';
+import { useTimedTextTrack } from 'data/stores/useTimedTextTrack';
+import { initVideoWebsocket } from 'data/websocket';
+import { theme } from 'utils/theme/theme';
+import { InstructorDashboardVODWidgetGeneralTitle } from './widgets/InstructorDashboardVODWidgetGeneralTitle';
+
+const StyledLiveVideoInformationBarWrapper = styled(Box)`
+  -webkit-box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+  box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+`;
+
+export const InstructorDashboardVOD = () => {
```

Comment:
```suggestion
export const DashboardVOD = () => {
```

Reply:
TODO : adopt a proper organization for widgets